.gitlab-ci.py

```diff
@@ -16,13 +17,18 @@
     python.pytest(),
     python.mypy("gcip"),
     python.bdist_wheel(),
+)
+
+dcc = DockerClientConfig(config_file_path="/kaniko/.docker")
```

Comment:
I do not know, what exactly you mean, `/kaniko/.docker` is the default config file path for `kaniko`.

Reply:
But you only know that, because you have read the kaniko documentation.
After your PR everybody using `gcip.addons.container.jobs.kaniko.execute` needs to know that.

Your change makes calling the kaniko.execute-Job more complicated.

Reply:
What I would prefer is, that

- DockerClientConfig allows to set the `config_file_path` after initialization
- The user can then create the DockerClientConfig without `config_file_path` and pass that object to the kaniko job.
- The job sets the `config_file_path` on its own.

Because the gcip should make it easy for the user to execute a kaniko job. The user should not be forced to read the kaniko documentation on how to configure the docker config for kaniko.

Reply:
Ok I understand. Maybe is this worth an issue. The DockerClientConfig class does not have the needed methods available. I want to keep this PR as clean as possible.

Reply:
I will add the necessary methods to DockerClientConfig in this PR, makes migration a bit smoother. ;-)
I also will add a functionality to allow kankio.execute() function to modify the config_file_path and config_file_name of `dcc` if not set by the user.

Reply:
Have you any opinion about how to implement this?
One or two method's? Method name suggestions?

Reply:
- One method seems to be easier and should keep the classes interface clean as possible.
- Suggestions:
  - `set_config_file_path` (most preferred one, because "set the path to that file" here)
    - 'set_config_file_path' sounds so obvious to me, that I have deleted all other odd suggestions

Reply:
Ok, I have no complains about this suggestion. Just another question. 
Right now, and I do not want to change this. At the moment you can set the path to the `config.json` e.g. `/foo/bar/baz/` and the filename `config.json` in the `__inti__` method 
`def __init__(self, config_file_path: str = "$HOME/.docker", config_file_name: str = "config.json") -> None:`

IMHO, I want to keep this behavior.  
Implementation suggestion:
`def set_config_file_path(self, path="$HOME/.docker", file_name="docker.json") -> None: `